### ISSUE TITLE
Driver isn't updated automatically after its version update

### ISSUE THREAD

**Issue by uslss** [dbeaver/dbeaver#17893]
Today I have upgraded DBeaver to 22.2.2.202210092258 (info from Help | About).

1. Database | Driver Manager.
2. Select DB2 | DB2 LUW and Edit button.
3. Libraries tab and click on Download/Upgrade.
![image](https://user-images.githubusercontent.com/10895030/195290920-550c10dd-b109-4d38-b51b-e872785af14d.png)
and dialog opens:
![image](https://user-images.githubusercontent.com/10895030/195291097-94f242a6-2f08-41c0-b76d-a20313ed35da.png)

This is obviously still and old driver version 11.5.0.0 (the problem). But I expect to get 11.5.7.0 like fix in this problem report should provide and milestone 22.2.2 promises.

Then I can continue and download the proposed driver and then connect to some Db2 database and test the JDBC driver version with SQL:
```
SELECT
CLIENT_PRDID
AS JDBC_VERSION
FROM
TABLE(SYSPROC.MON_GET_CONNECTION(SYSPROC.MON_GET_APPLICATION_HANDLE(), -1))

```

and SQL returns: JCC04260 so JDBC v4.26.x which is according to official JDBC driver web page: https://www.ibm.com/support/pages/db2-jdbc-driver-versions-and-downloads still three years old  JDBC driver for v11.5.0.0.

_Originally posted by @igor2x in https://github.com/dbeaver/dbeaver/issues/17427#issuecomment-1275791760_

**Comment by serge-rider** [comment]
We don't update driver version automatically now. It is a new feature request, not a bug.

**Comment by serge-rider** [comment]
To enable automatic driver update you need to turn on option "Check for new driver versions" in driver preferences.
![image](https://user-images.githubusercontent.com/6398845/195476609-34172b72-a30d-4862-ac13-b35f6bfe1904.png)

**Comment by igor2x** [comment]
@serge-rider I tried setting the preference and driver does not update. But...

I actually don't care if JDBC driver gets automatically updated or not, this was @uslss nice gesture to open new bug report, but unfortunately my previous bug report https://github.com/dbeaver/dbeaver/issues/17427 was closed down.

What I care is what I have written in my original bug report comment: Dbeaver v22.2.2 should provide new Db2 JDBC driver. Like it is seen from print-screens in this report (first post) I have **manually** tried to update the JDBC driver and there was no update available.

Like I see @uslss conclusion was that driver does not get automatically updated, but my complain is that Db2 JDBC driver should be available in Dbeaver 22.2.2, but manually update the driver and it does not update. I assume, something was wrong with fix that did not provide new JDBC driver.

**Comment by E1izabeth** [comment]
@igor2x the current fix will be available in the next release.

As a temporary workaround you can manually delete the corresponding cached record about the driver in .metadata\\.config\drivers.xml to force DBeaver to use updated driver version.
